app/mailers/comment_notification_mailer.rb

```diff
@@ -1,22 +1,39 @@
 class CommentNotificationMailer < ApplicationMailer
-  def email_notification(comment)
+
+  #email_notification
```

Comment:
Let's remove this comment and the one below for speaker_notification.  I don't see it adding anything valuable.